environments/kolla/files/overlays/keystone/wsgi-keystone.conf

```diff
@@ -54,6 +54,7 @@ LogLevel info
 {% endif %}
 
 {% if keystone_enable_federation_openid %}
+    OIDCXForwardedHeaders X-Forwarded-Proto
```

Comment:
Maybe as an idea something like this:
{% if keystone_oidc_forward_header is defined %}
    OIDCXForwardedHeaders "{{ keystone_oidc_forward_header }}"
{% endif %}

and in the environment/kolla/configuration.yml
under  section keystone

keystone_oidc_forward_header: "X-Forwarded-Proto"

Reply:
Changes done. Let's see the CI results.